<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QWidget" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>155</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>150</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="leUpDownStep">
              <property name="minimumSize">
               <size>
                <width>15</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>5</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="sliderUpDown">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="pageStep">
               <number>5</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>2</number>
            </property>
            <item row="1" column="1">
             <widget class="QPushButton" name="pbHome">
              <property name="text">
               <string>Home</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pbLeft">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>⬅</string>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pbUp">
              <property name="text">
               <string>⬆</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="pbRight">
              <property name="text">
               <string>➡</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pbDown">
              <property name="text">
               <string>⬇</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pbLeftUp">
              <property name="text">
               <string>↖</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="pbRightUp">
              <property name="text">
               <string>↗</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pbLeftDown">
              <property name="text">
               <string>↙</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="pbRightDown">
              <property name="text">
               <string>↘</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>40</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="leLeftRightStep">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>10</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderLeftRight">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>20</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_7" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="pbPatrol">
           <property name="toolTip">
            <string>Patrol</string>
           </property>
           <property name="text">
            <string>巡邏</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbPatrol">
           <property name="text">
            <string>continue</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
